cwl/src/main/scala/cwl/CommandLineTool.scala

```diff
@@ -470,101 +358,12 @@ object CommandLineTool {
                                     id: String,
                                     label: Option[String] = None,
                                     secondaryFiles: Option[SecondaryFiles] = None,
-                                    format: Option[Expression :+: Array[String] :+: String :+: CNil] = None, //only valid when type: File
+                                    format: Option[Expression :+: String :+: Array[String] :+: CNil] = None, //only valid when type: File
```

Comment:
why?

Reply:
Because it now extends `InputParameter` which defines an abstract member for `format`, so the types have to be in the same order otherwise it doesn't compile.
Which actually reminds me that I created a`type Format` so I'll just use that here